tests/Install-Scripts.Test/GivenThatIWantToGetTheSdkLinksFromAScript.cs

```diff
@@ -91,6 +94,32 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
             commandResult.Should().HaveStdOutContainingIgnoreCase(runtimeType);
         }
 
+
+        [Theory]
+        [InlineData("3.1", _feedCredential)]
+        [InlineData("5.0.2xx", _feedCredential)]
+        [InlineData("LTS", "?" + _feedCredential)]
+        [InlineData("release/1.0.0", "?" + _feedCredential)]
+        [InlineData("master", _feedCredential)]
+        public void WhenFeedCredentialParameterIsPassedToInstallScripts(string channel, string feedCredential)
+        {
+            var args = new List<string> { "-dryrun", "-channel", channel, "-feedcredential", feedCredential, "-verbose" };
+
+            var commandResult = CreateInstallCommand(args)
+                            .CaptureStdOut()
+                            .CaptureStdErr()
+                            .Execute();
+
+            //  Standard 'dryrun' criterium
+            commandResult.Should().Pass();
+            commandResult.Should().NotHaveStdOutContaining("dryrun");
+            commandResult.Should().HaveStdOutContaining("Repeatable invocation:");
+
+            // Checking format of the link after applying feed credential
+            string linkWithCredentialsExpression = @"Primary named payload URL: [a-zA-Z0-9:/.-]*\?" + feedCredential;
+            commandResult.Should().HaveStdOutContaining(output => Regex.IsMatch(output, linkWithCredentialsExpression));
```

Comment:
We have two links printed, is that right? One for primary and one for legacy. If legacy one works and primary one is broken, the test will pass today. It would be nice if both were checked separately.

Reply:
We do not always have two links printed, the guaranteed one is the primary one. The legacy one is printed if it is not empty and it happens sometimes. So, we need either to copy the behavior of the scripts and compute a condition indicating whether we have the legacy link, or do not put much of an effort here and just check only the primary link. I think the check of the primary link with the credentials is currently enough.